### ISSUE TITLE
find a better comparative dataset

### ISSUE THREAD

**Issue by mwartell** [twosixlabs/armory-library#79]
We landed on the EuroSAT dataset for comparative analysis of the convnext, vit, and swin models because we noticed that they had different accuracy metrics across models. This generated our first by-model comparison. Unfortunately, the EuroSAT data set is poorly structured for our work and takes hours to download onto new evaluation machines.

While we could mitigate this annoyance with mechanisms like 

- cache the dataset metadata on a shared drive
- repackage the dataset into something like parquet
- 
it is not clear that the benefits for this one dataset would be worth the effort. We've been hanging onto that dataset because we accidentally hit a difference. There are other models and datasets out there some which may be better suited to our needs.

In anticipation of the possible VisDrone requirement noted in #75, we have found a [Papers With Code page](https://paperswithcode.com/sota/object-detection-on-visdrone-det2019-1) which refers to at least 3 models trained for that set. 

I am looking for an image classification task for which we can find multiple published models. If they have different non-attacked performance  in advance of our trial, even better,

**Comment by mwartell** [comment]
Torchvision provides many pretrained models as listed under [Table of all available classification weights](https://pytorch.org/vision/stable/models.html#table-of-all-available-classification-weights) the top of which is shown here

![image](https://github.com/twosixlabs/armory-library/assets/462695/75449a08-cc91-4e3f-a35d-3b2fee8a89b9)

Notably, they in both Accuracy and "GFLOPS" which I presume is a measure of computational cost for the benign case that they run. 

Running on Torchvision "toy" problem could provide some exploratory convenience for comparative analysis.

**Comment by Honaker** [comment]
I did create a better parquet packaged dataset on HuggingFace that is still available https://huggingface.co/datasets/Honaker/eurosat_dataset
